tests/integration/test_data_cli.py

```diff
@@ -81,6 +81,52 @@ def test_data_search_quick_filter_success(invoke, item_types):
     assert len(result.output.strip().split('\n')) == 1  # we have 1 feature
 
 
+@respx.mock
+@pytest.mark.parametrize("limit,limited_list_length", [(None, 100), (0, 102),
```

Comment:
This is such a great test. Nice work on the parametrization.

Reply:
I also adapted this straight from another test you've written haha thank you for setting a great example!